src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -295,13 +375,44 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
                     }
                 }
 
-                _installedPackages = vmInstalledPackages.OrderBy(p => p.Name).ToList();
+                vmInstalledPackages = vmInstalledPackages.OrderBy(p => p.Name).ToList();
+
+                await UpdateLoadedPackages(vmInstalledPackages);
+                _installedPackages = vmInstalledPackages;
             }
+        }
+
+        private async Task ReloadLoadedPackagesAsync() {
+            await TaskUtilities.SwitchToBackgroundThread();
+            try {
+                var currentLoadedPackages = _loadedPackages;
+                var currentInstalledPackages = _installedPackages;
+                var loadedPackageNames = (await _packageManager.GetLoadedPackagesAsync()).OrderBy(n => n).ToList();
+
+                if (loadedPackageNames.Equals(currentLoadedPackages, (n, p) => n.EqualsIgnoreCase(p.Name))) {
+                    return;
+                }
+
+                await UpdateLoadedPackages(currentInstalledPackages, loadedPackageNames);
+                _coreShell.DispatchOnUIThread(() => {
+                    if (_selectedTab == SelectedTab.LoadedPackages) {
+                        IsLoading = false;
+                        ReplaceItems(_loadedPackages);
+                    }
+                });
+            } catch (RPackageManagerException ex) {
```

Comment:
Do you need to catch OperationCanceledException here, or is it handled by a caller?